modules/CmpdReg/src/client/src/Parent.js

```diff
@@ -32,7 +32,7 @@ $(function() {
 					exactMass: js.exactMass,
                     molFormula: js.molFormula,
 					comment: js.comment,
-                    chemist: new PickList(js.chemist),
+                    chemist: js.chemist,
```

Comment:
I'm confused about how a PickList with the current chemist selected is getting replaced with just the raw chemist code. Could you clarify @brianbolt ?

Reply:
The picklist, which was really just the chemist object was required previously as it contained the id and other attributes required by ROO to match the chemist back to the scientist table.  We are now just passing the string along.